wolfSSL/src/chapter02.md

```diff
@@ -891,6 +1800,15 @@ Compiles in a faster mod_exp implementation at the expense of code size.
 
 Disable printing of leading zero in hexidecimal string output. For example, if this macro is defined, the value 8 will be printed as the string "0x8" but if it is not defined it will be printed as "0x08". Defining this macro can reduce code size.
 
+#### WC_ASN_NAME_MAX
+
+Used when calculating curve x509. Use value from asn.h.
```

Comment:
This allows overriding the maximum name support for an X.509 certificate field. "curve x509" is not correct